supervisor/host/info.py

```diff
@@ -66,6 +67,11 @@ async def get_dmesg(self) -> bytes:
 
         return stdout
 
+    def get_disk_free_space(self, path: str = "/data") -> float:
```

Comment:
Is `/data` the path that is used for pulling docker images?

Reply:
Maybe use this property instead?
https://github.com/home-assistant/supervisor/blob/e1ce19547e0a0d056d645436aa291e86fbabbe9b/supervisor/config.py#L135-L138